content/slides/open-science-and-data.md

```diff
@@ -72,11 +86,36 @@ title-slide-attributes:
 :::
 
 
+## Communication is key
+
+If you are actively engaged in research in the Arctic, make yourself familiar
+with reccomended communication channels and programs for interfacing with local
+communities.
+
+::: {.r-fit-text}
+* [Issaffik Arctic Gateway](https://isaaffik.org/): user driven web platform
+  supporting Arctic research and collaboration.
+* [Arctic Hub](https://arctichub.gl/): a space where researchers can connect,
+  institutions can collaborate, and citizens can engage in Greenlandic research.
+* [Navigating the New Arctic Community Office](https://nna-co.org/): building
+  awareness, partnerships, opportunities, and resources for collaboration and
+  equitable knowledge generation within, between, and beyond the research
+  projects funded by the [NSF] [NNA
+  Initiative](https://www.nsf.gov/news/special_reports/big_ideas/arctic.jsp).
```

Comment:
How can participants engage with each of these? Register an account and make forum posts? Participate in an email list?

Reply:
I think just getting these resources in front of them as a resource to explore is the place to start!